### PR TITLE
Add gradle plugin doc

### PR DESCRIPTION
**Pull request by acbeni**
Since our work is taking shape it's high time we start documenting i
This PR tries to add the gradle plugin doc

### CODE REVIEW

File: tools/codegen-gradle-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+### VRAP Gradle plugin
+
+In order to make the integration of the code generator in a gradle build easy we created a gradle plugin with id
+`io.vrap.rmf.codegen-plugin` (not yet published to gradle central), in order to use it you have to specify the `uri`
+parameter with is the raml file location, plus targets
+
+
+A target is a generation target it contains
+
+   * serviceName: your service name
```
Comment (katmatt) [inline tools/codegen-gradle-plugin/README.md]: What is a service name? May be we should call it `targetName`?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -3,4 +3,5 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+org.gradle.daemon=false
```
Comment (katmatt) [inline gradle/wrapper/gradle-wrapper.properties]: Do we really want to disable the daemon for all of us? Which problems did it cause for you?

Comment (acbeni) [reply gradle/wrapper/gradle-wrapper.properties]: here it just causes some locking problems
but its fine for me to reenable it

File: tools/codegen-gradle-plugin/README.md (hunk 1 above)
Comment (acbeni) [reply tools/codegen-gradle-plugin/README.md]: service name means actually the module name for example `import-storage-service`....

**Comment by katmatt** [comment]
Looks good to me 👍

**Pull request merged**